src/database/migrations/20191023171009-request-notification.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.createTable('Notifications', {
+      id: { allowNull: false,autoIncrement: true, primaryKey: true, type: Sequelize.INTEGER },
+      RequestId: {
+        type: 'integer',
+        onDelete: 'CASCADE',
+        references: {
+          model: 'Requests',
+          key: 'id',
+        },
+      },
+      managerId: { type: Sequelize.INTEGER, allowNull: false },
+      createdAt: { allowNull: false, type: Sequelize.DATE, defaultValue: Sequelize.NOW },
+      updatedAt: { allowNull: false, type: Sequelize.DATE},
+    });
+  },
+  down: (queryInterface, Sequelize) => {
+    return queryInterface.dropTable('Notifications');
+  }
+};
```

Comment:
great work! However, I don't understand the purpose of this table since we already have the notification table. I think once a request is stored in the notification table, the manager as a user will be able to see it when he logs in!

Reply:
This PR is currently closed because there is another one it depends on.

Reply:
great! I didn't know!